src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import { Text, FlatList } from 'react-native';
```

Comment:
is formatting broken? this has single quotes when the file above has double quotes

Reply:
prettified :)